package.json

```diff
@@ -1,98 +1,84 @@
 {
   "name": "@zondax/ledger-cosmos-js",
-  "author": "Zondax AG",
-  "license": "Apache-2.0",
   "version": "4.0.0",
   "description": "Node API for Cosmos App (Ledger Nano S/S+/X)",
-  "main": "./dist/index.js",
-  "typings": "./dist/index.d.ts",
-  "types": "./dist/index.d.ts",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/cosmos/ledger-cosmos-js.git"
-  },
   "keywords": [
     "Ledger",
     "Typescript",
     "Tendermint",
     "Cosmos"
   ],
+  "homepage": "https://github.com/Zondax/ledger-cosmos-js",
   "bugs": {
-    "url": "https://github.com/cosmos/ledger-cosmos-js/issues"
+    "url": "https://github.com/Zondax/ledger-cosmos-js/issues"
   },
-  "homepage": "https://github.com/cosmos/ledger-cosmos-js",
-  "resolutions": {
-    "@ledgerhq/hw-transport": "6.27.1",
-    "@ledgerhq/devices": "6.27.1",
-    "@ledgerhq/hw-transport-node-hid": "6.27.1",
-    "@ledgerhq/hw-transport-node-hid-noevents": "6.27.1"
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cosmos/ledger-cosmos-js.git"
+  },
+  "license": "Apache-2.0",
+  "author": "Zondax AG",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "typings": "./dist/index.d.ts",
+  "files": [
+    "dist/**",
+    "LICENSE",
+    "package.json"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "format": "FORCE_COLOR=1 prettier --write . && sort-package-json",
+    "format:check": "FORCE_COLOR=1 prettier --check .",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix .",
+    "test": "yarn build && jest .",
+    "upgrade": "bunx npm-check-updates -i"
   },
   "dependencies": {
-    "@ledgerhq/hw-transport": "6.30.6",
-    "@zondax/ledger-js": "^0.10.0",
-    "bech32": "^2.0.0",
+    "@ledgerhq/hw-transport": "6.31.2",
+    "@noble/hashes": "^1.4",
```

Comment:
Are we sure this is browser compatible? We are changing from browserfy to this version.